flake.nix

```diff
@@ -1,30 +1,32 @@
 {
-  description = "jujutsu";
+  description = "Jujutsu VCS, a Git-compatible DVCS that is both simple and powerful";
 
   inputs = {
+    # For listing and iterating nix systems
+    flake-utils.url = "github:numtide/flake-utils";
+
     # For installing non-standard rustc versions
     rust-overlay.url = "github:oxalica/rust-overlay";
     rust-overlay.inputs.nixpkgs.follows = "nixpkgs";
+    rust-overlay.inputs.flake-utils.follows = "flake-utils";
   };
 
-  outputs = { self, nixpkgs, rust-overlay, ... }:
+  outputs = { self, nixpkgs, flake-utils, rust-overlay }:
     let
-      lib = nixpkgs.lib;
-      systems = [
-        "aarch64-linux"
-        "aarch64-darwin"
-        "i686-linux"
```

Comment:
Is `i686-linux` here on purpose?

I'm asking, because it's the only difference to the list of default systems. Without it we wouldn't have an explicit list at all.

Reply:
Do you remember, @pingiun?

Reply:
I believe I just copied this list from one of my own projects. Using a predefined default list is fine, as long as it contains all of the platforms `jj` wants to support